app/serializers/auction_serializer.rb

```diff
@@ -43,6 +43,15 @@ def winning_bid
   private
 
   def find_winning_bid
-    RulesFactory.new(object).create.winning_bid
+    if auction_status.available?
+      NullBid.new
+    else
+      RulesFactory.new(object).create.winning_bid
```

Comment:
benefit of factory class is that it brings all of the logic into one confined thing rather than adding to the "API" of an existing object, like `Auction` or `AuctionPresenter`...but yeah. Still need to think through how we're dealing with all of these view / presenter / model layers